ddtrace/tracer/option.go

```diff
@@ -535,6 +539,14 @@ func newConfig(opts ...StartOption) *config {
 	globalTagsOrigin := c.globalTags.cfgOrigin
 	c.initGlobalTags(c.globalTags.get(), globalTagsOrigin)
 
+	// Check if CI Visibility mode is enabled
+	if internal.BoolEnv(constants.CiVisibilityEnabledEnvironmnetVariable, false) {
```

Comment:
Typo in constant's name :)

```suggestion
	if internal.BoolEnv(constants.CiVisibilityEnabledEnvironmentVariable, false) {
```

Reply:
🤦🏻 fixed in [c184be3](https://github.com/DataDog/dd-trace-go/pull/2739/commits/c184be335ef35bdb9fd4884f17f6191935ca2c6f)